src/get-options-overrides.ts

```diff
@@ -16,13 +16,26 @@ export function getOptionsOverrides({ useTsconfigDeclarationDir, cacheRoot }: IO
 		noEmitOnError: false,
 		inlineSourceMap: false,
 		outDir: normalize(`${cacheRoot}/placeholder`), // need an outdir that is different from source or tsconfig parsing trips up. https://github.com/Microsoft/TypeScript/issues/24715
-		moduleResolution: tsModule.ModuleResolutionKind.NodeJs,
 		allowNonTsExtensions: true,
+		moduleResolution: tsModule.ModuleResolutionKind.Node10,
```

Comment:
mmm, I think it would be more straightforward logic to have _no_ override by default. we would only add an override in the case of `classic`

Reply:
Yeah, if we pass bundler through the only problem could be new types in the future, but we can deal with it in readme.

Reply:
Or we could add a check similar to the `module` one.
But I'm not particularly worried about forward compat, future changes may require future changes after all.

This line still needs to be removed